lib/commons/aria/required-owned.js

```diff
@@ -9,15 +9,13 @@ import lookupTable from './lookup-table';
  * @return {Mixed} Either an Array of required owned elements or `null` if there are none
  */
 function requiredOwned(role) {
-	'use strict';
-	let owned = null;
-	const roles = lookupTable.role[role];
+	const roleDef = standards.ariaRoles[role];
 
-	if (roles) {
-		// TODO: es-module-utils.clone
-		owned = axe.utils.clone(roles.owned);
-	}
-	return owned;
+  if (!roleDef || !Array.isArray(roleDef.requiredOwned)) {
+    return null;
+  }
+
+  return [...roleDef.requiredOwned];
```

Comment:
Think your editor went wrong:

```suggestion
	 if (!roleDef || !Array.isArray(roleDef.requiredOwned)) {
	 	 return null;
	}

	 return [...roleDef.requiredOwned];
```